bin/deploy

```diff
@@ -45,13 +42,14 @@ usage() {
   # note that heredocs are prefixed with tab chars
   fail "$(cat <<-EOF
 
-		Usage: $0 [-2] [-3] [-b] [-h]
+		Usage: $0 [-2] [-3] [-b] [-h] [-r]
 
 		Specific options:
-		    -b          use bleeding edge conda packages
-		    -3          use Python 3 (default)
-		    -2          use Python 2 (no longer supported -- fatal error)
-		    -h          show this message
+	    -b          use bleeding edge conda packages
```

Comment:
We might want to tweak the message here. `-b` means that the conda package list as defined in the `-r REF` (or default) is used without version constraints.  That list of packages can be different depending on the choice of `-r`.
How about:

> install conda packages without version constraints

?

Reply:
Fine for me. 
Maybe `Install environment using bleed packages`.
However, if we want to improve this aspected now, some logic should be added to have a different environment name in this case. This will make obsolete DM-17823.